### ISSUE TITLE
should the hints flash in the collection area if an expression is correct but not reduced?

### ISSUE THREAD

**Issue by jbphet** [phetsims/expression-exchange#60]
I'm currently working on the game implementation, and it strikes me as a potential usability issue that on the first four levels the user can create an expression that looks visually like the expression hint but is rejected as an answer because it hasn't been reduced.  I suggested to @amanda-phet that perhaps the dotted circle "hints" in the collection area should flash as a way to tell the user, for example, "you're close, but it should have 2 terms instead of 3".  @amanda-phet thought that this might be worthwhile, but suggested that we wait until initial interviews have been done on the game before taking the time to implement this feature.

For reference, here is a screenshot of what I'm describing, i.e. an expression that visually matches the description but is rejected due to it being un-reduced.

![exex-rejection](https://cloud.githubusercontent.com/assets/3082831/23627356/ab599cb0-026d-11e7-90b9-c4529ba7bfbb.PNG)

**Comment by jbphet** [comment]
Leaving unassigned until we start interviews on the game.

**Comment by jbphet** [comment]
Assigning to @amanda-phet since she said that she would decide about this based on the outcome of interviews.

**Comment by amanda-phet** [comment]
Based on the game flow I observed in interviews, I am suggesting that we make changes to levels 1-3. What I noticed is that students figured out the cues quite easily and learned to simplify their answers. When they got to levels 4-8 students continue to simplify their answers even though they didn't need to. However, I'd like students to know they can leave the answer unsimplified, and see that their answer might not look identical to the target but it is still equivalent. One way of resolving this is by changing levels 1-3:  

* Targets will be "expressions" instead of "collections." 
* Remove the cues to simplify answers 
* Do not require user to simplify answers

<img width="890" alt="screen shot 2017-04-05 at 12 05 59 pm" src="https://cloud.githubusercontent.com/assets/7852916/24719801/4866045c-19f8-11e7-9a54-504a7e754f87.png">
